task2.py

```diff
@@ -0,0 +1,3 @@
+text = 'MIPT Students'
+n = int(input())
+print('Hello, ' + ('{}, '.format(text))*(n-1) + '{}'.format(text))
```

Comment:
тут можно использовать format string и второй раз не понятно зачем форматировать
```suggestion
print('Hello, ' + (f'{text}, ')*(n-1) + text)
```